org.eclipse.xtext/src/org/eclipse/xtext/documentation/impl/MultiLineJavaDocTypeReferenceProvider.java

```diff
@@ -44,6 +64,44 @@
 		return regions;
 	}
 
+	/**
+	 * Returns the referenced element in javadoc comment at a given <i>offset</i>.
```

Comment:
I would like to see some documentation specifying that the return value can be null.